frontend/src/components/hub/HubHeadlineContainer.tsx

```diff
@@ -17,9 +18,9 @@ const useStyles = makeStyles((theme) => ({
 
     maxWidth: "800px",
     borderRadius: 5,
-    border: theme.borders.thick,
+    border: `3px solid ${theme.palette.primary.main}`,
```

Comment:
Again, curious how did you determine this?

Reply:
Same as above, just directly using the css code rather than calling it from the theme